kolibri/utils/options.py

```diff
@@ -198,7 +198,12 @@ def language_list(value):
             "type": "string",
             "default": "content",
             "envvars": ("KOLIBRI_CONTENT_DIR",),
-        }
+        },
+        "CONTENT_FALLBACK_DIRS": {
+            "type": "string",
```

Comment:
Cool, thx -- done [here](https://github.com/learningequality/kolibri/pull/6865/commits/dd39641184627fd5b5bc7da9297c3071a6953097).

I didn't want to enforce path string formats using a regex or whatnot, because that can be hairy, from what I read (with Unicode, etc). Nor do I want to enforce that they already exist...

Reply:
Ah, it was the latter that I was more interested in - but the idea here is to allow paths to come into being without a restart of Kolibri?

Reply:
The idea is that, for example: if a removable drive isn't currently plugged in/loaded yet, it doesn't block Kolibri from starting. Or that there could be a list of "potential" paths in here, but we shouldn't blow up just because one doesn't exist (just like PATH, PYTHONPATH, etc, are tolerant to nonexistent paths).